test/CryptoAxus.Application.Features.Artist/GetArtistByWalletAddress/Handler/GetArtistByWalletAddressHandlerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace CryptoAxus.Application.Features.Artist.GetArtistByWalletAddress.Handler;
+
+public class GetArtistByWalletAddressHandlerTests : GetArtistByWalletAddressTestData
+{
+    [Fact]
+    public async Task When_Artist_WalletAddress_Is_Passed_Expect_Artist_Document_Is_Returned()
```

Comment:
Add _ between WalletAddress, should be Wallet_Address

Reply:
Resolved